austrakka/utils/helpers/analysis.py

```diff
@@ -3,7 +3,4 @@
 
 
 def get_analysis_by_abbrev(abbrev: str):
-    response = api_get(
-        path=f"{ANALYSIS_PATH}/abbrev/{abbrev}"
-    )
-    return response['data'] if ('data' in response) else response
+    return api_get(path=f"{ANALYSIS_PATH}/abbrev/{abbrev}")['data']
```

Comment:
Can 'data' be unpacked by api_get()?

Reply:
No because we have logic that reads the `messages` object too